### ISSUE TITLE
Error when running sails lift

### ISSUE THREAD

**Issue by giltig** [balderdashy/sails-mongo#367]
TypeError: Cannot read property 'close' of undefined
    at I:\sametrip\node_modules\sails-mongo\lib\adapter.js:138:13
    at I:\sametrip\node_modules\sails-mongo\node_modules\async\lib\async.js:187:20
    at I:\sametrip\node_modules\sails-mongo\node_modules\async\lib\async.js:239:13
    at _arrayEach (I:\sametrip\node_modules\sails-mongo\node_modules\async\lib\async.js:91:13)
    at _each (I:\sametrip\node_modules\sails-mongo\node_modules\async\lib\async.js:82:13)
    at Object.async.forEachOf.async.eachOf (I:\sametrip\node_modules\sails-mongo\node_modules\async\lib\async.js:238:9)
    at Object.async.forEach.async.each (I:\sametrip\node_modules\sails-mongo\node_modules\async\lib\async.js:215:22)
    at Object.module.exports.adapter.teardown (I:\sametrip\node_modules\sails-mongo\lib\adapter.js:137:22)
    at C:\Users\gil\AppData\Roaming\npm\node_modules\sails\lib\hooks\orm\index.js:244:19
    at C:\Users\gil\AppData\Roaming\npm\node_modules\sails\node_modules\async\lib\async.js:122:13
    at _each (C:\Users\gil\AppData\Roaming\npm\node_modules\sails\node_modules\async\lib\async.js:46:13)
    at Object.async.each (C:\Users\gil\AppData\Roaming\npm\node_modules\sails\node_modules\async\lib\async.js:121:9)
    at Sails.hook.teardown (C:\Users\gil\AppData\Roaming\npm\node_modules\sails\lib\hooks\orm\index.js:241:13)
    at Sails.g (events.js:260:16)
    at emitNone (events.js:67:13)
    at Sails.emit (events.js:166:7)

**Comment by elwadi** [comment]
I had this error, and now i use  package to sails-mongo@0.11.1

**Comment by jodonnell-broadsoft** [comment]
this version worked for me as well

**Comment by sailsbot** [comment]
Thanks for posting, @giltig.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
